discovery-service/src/main/java/com/ca/mfaas/discovery/config/HttpWebSecurityConfig.java

```diff
@@ -70,7 +65,6 @@ public void configure(WebSecurity web) {
 
     @Override
     protected void configure(HttpSecurity http) throws Exception {
-        log.warn("Discovery service is configured to use insecure HTTP protocol");
```

Comment:
Do you think that this message is not needed?

Reply:
yes it's not needed because there is one in HTTPS factory saying the same thing for all services